usaspending_api/api_contracts/contracts/v2/awards/award_id.md

```diff
@@ -769,6 +843,10 @@ This endpoint returns a list of data that is associated with the award profile p
 + `place_of_performance` (required, Location, fixed-type)
 + `executive_details` (required, Executive, fixed-type)
 + `cfda_info` (required, array[CFDAInfo], fixed-type)
++ `total_account_outlay` (required, number)
++ `total_account_obligation` (required, number)
++ `account_obligations` (required, array[object], fixed-type)
++ `account_outlays` (required, array[object], fixed-type)
```

Comment:
Usually we define the actual object and reference that definition rather than use generic objects.  Also, should  these mention that they're obligations/outlays broken down by defc?

Reply:
`by` IMO indicates the value is a map. `defc_account_outlays`. I really don't care what it's named TBH.